src/index.ts

```diff
@@ -604,6 +614,12 @@ export interface AcrolinxSidebar {
    * @param message The message to show.
    */
   showMessage(message: Message): void;
+
+  /**
+   * Perform a search for suggestions with the given prefix .
+   * @param prefix The prefix on which to perform a search for suggestions.
+   */
+  reusePrefixSearch?(prefix: string): void;
```

Comment:
Since it is not a prefix search anymore, we should name this function differently. I would call it "startReuseSearch"

Reply:
Updated to "liveSearch"